src/components/StudentProfile.js

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import axios from 'axios';
+import * as actions from '../redux/actions';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Card } from 'react-bootstrap';
+
+import Whiteboard from '../images/Whiteboard.jpeg';
+import Khan from '../images/Khan-Academy.png';
+import Classroom from '../images/google-classroom.jpg';
+import Facebook from '../images/VBB-FB.png';
+import Meet from '../images/google-meet.png';
+import Website from '../images/VBB-Website.png';
+
+class Student extends React.Component {
+  state = {
+    sessionSlots: [],
+  };
+
+  fetchSessionSlotData = () => {
```

Comment:
Any API calls should be moved to Redux and connect this component. Not sure if this is an iterative PR though :)